src/common/models/essence/essence.ts

```diff
@@ -487,23 +487,46 @@ export class Essence extends ImmutableRecord<EssenceValue>(defaultEssence) {
     return this.changeFilter(filter.getSpecificFilter(timekeeper.now(), dataCube.getMaxTime(timekeeper), timezone));
   }
 
+  private defaultSplitSort(split: Split): Sort {
+    const { dataCube, series } = this;
+    const dimension = dataCube.getDimension(split.reference);
+    const { sortStrategy, name, kind } = dimension;
+    if (sortStrategy === "self" || sortStrategy === name) {
+      return new DimensionSort({ reference: name, direction: SortDirection.ascending });
+    }
+    if (sortStrategy && series.hasMeasureSeries(sortStrategy)) {
+      return new SeriesSort({ reference: sortStrategy, direction: SortDirection.descending });
+    }
+    if (kind === "time") {
```

Comment:
Just "time" or continuous? I vote for time. Numeric dimensions with limits are tricky.

Reply:
IMHO only time is so specific to set default sort ascending by dimension.